src/pybind/mgr/dashboard/frontend/src/app/shared/components/helper/helper.component.html

```diff
@@ -0,0 +1,11 @@
+<ng-template #popoverTpl>
+  <div [innerHtml]="html"></div>
+  <ng-content></ng-content>
+</ng-template>
+<i class="fa fa-question-circle"
+   aria-hidden="true"
+   [popover]="popoverTpl"
+   placement="bottom"
+   container="body"
+   [outsideClick]="true">
```

Comment:
Is property binding really necessary here?

Reply:
Fixed

Reply:
This should be a property binding. 

Otherwise angular will not interpret the value and will  see it as a string and not a boolean.
p.e.: If you set ``outsideClick="false"`` popover component will think the value is actually true, because it is a string and not empty.

Only situations when we should remove the binding is if the value is actually a string and it will never change (Or you know the component will convert the string to the correct type).

Reply:
@tspmelo I've tested it and you are right.Thanks.

@votdev I've reverted the change, so I'm using property binding again.